VAMobile/src/utils/secureMessaging.tsx

```diff
@@ -72,11 +73,29 @@ export const getMessagesListItems = (
       },
     ]
 
+    const folder = (): string => {
```

Comment:
I noticed this code block repeated a few times, but is it needed? Since FolderNameTypeConstants looks like this:

```
{
  drafts: 'Drafts',
  deleted: 'Deleted',
  sent: 'Sent',
  inbox: 'Inbox',
}
```

Can't you just send the constant value in the event as is?

Reply:
doesn't cover custom folders and we don't care what their custom folder name is, just that they are using custom folders. So it needs to function this way as the constants doesn't cover this case.